sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -263,7 +263,7 @@ To build and install the starter use the following command line:
 ```
 
 
-# Azure KeyVault Certificates client library for Java
+# Azure Key Vault Certificates client library for Java
```

Comment:
We should not include these sections just so that the CI pipelines don't complain. I think that, for the sake of consistency, we should make it so that this README is in line with the one for the [Secrets Spring Boot Starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-keyvault-secrets). We can always include the information above in a section where it makes sense, even additional to the ones below if you think that's better.